frontend/packages/integration-tests-cypress/tests/storage/clone.spec.ts

```diff
@@ -0,0 +1,70 @@
+import { testName, checkErrors } from '../../support';
+import { listPage } from '../../views/list-page';
+import { PVC, testerDeployment } from '../../mocks/snapshot';
+import { detailsPage } from '../../views/details-page';
+import { modal } from '../../views/modal';
+import { PVCDetails } from '../../views/storage/clone';
+import { resourceStatusShouldContain } from '../../views/common';
+
+const cloneName = `${PVC.metadata.name}-clone`;
+const cloneSize = '2';
+
+describe('Clone Tests', () => {
+  before(() => {
+    cy.login();
+    cy.createProject(testName);
+    cy.exec(`echo '${JSON.stringify(PVC)}' | oc apply -n ${testName} -f -`);
+    cy.exec(`echo '${JSON.stringify(testerDeployment)}' | oc apply -n ${testName} -f -`);
+    cy.clickNavLink(['Storage', 'Persistent Volume Claims']);
+    listPage.filter.byName(PVC.metadata.name);
+    resourceStatusShouldContain('Bound');
+  });
+
+  afterEach(() => {
+    checkErrors();
+  });
+
+  after(() => {
+    cy.exec(`echo '${JSON.stringify(testerDeployment)}' | oc delete -n ${testName} -f -`);
+    cy.exec(`echo '${JSON.stringify(PVC)}' | oc delete -n ${testName} -f -`);
+    cy.deleteProject(testName);
+    cy.logout();
+  });
+
+  it('Creates PVC Clone', () => {
+    listPage.rows.clickKebabAction(PVC.metadata.name, 'Clone PVC');
+    modal.shouldBeOpened();
+    modal.submitShouldBeEnabled();
+    cy.byTestID('input-request-size')
+      .clear()
+      .type(cloneSize);
+    modal.submit();
+    modal.shouldBeClosed();
+    cy.location('pathname').should('include', `persistentvolumeclaims/${PVC.metadata.name}-clone`);
+    detailsPage.titleShouldContain(`${PVC.metadata.name}-clone`);
+    cy.exec(`oc get pvc ${PVC.metadata.name}-clone -n ${testName} -o json`)
+      .its('stdout')
+      .then((res) => {
+        const pvc = JSON.parse(res);
+        cy.get(PVCDetails.name).contains(pvc.metadata.name);
+        cy.get(PVCDetails.namespace).contains(pvc.metadata.namespace);
+        cy.byTestID('pvc-requested-capacity').contains(`${cloneSize} GiB`);
+      });
+  });
+
+  it('Lists Clone', () => {
+    cy.clickNavLink((['Persistent Volume Claims'] as unknown) as [string, string]);
```

Comment:
```suggestion
    cy.clickNavLink((['Persistent Volume Claims']);
```
This issue has been fixed already.